pkg/awsutils/awsutils.go

```diff
@@ -445,21 +437,20 @@ func (cache *EC2InstanceMetadataCache) refreshVPCIPv4CIDRs(mac string) error {
 
 	newVpcIPv4CIDRs := StringSet{}
 	newVpcIPv4CIDRs.Set(vpcIPv4CIDRs)
-	addedVpcIPv4CIDRs   := newVpcIPv4CIDRs.Difference(&cache.securityGroups)
-	deletedVpcIPv4CIDRs := cache.securityGroups.Difference(&newVpcIPv4CIDRs)
+	addedVpcIPv4CIDRs := newVpcIPv4CIDRs.Difference(&cache.vpcIPv4CIDRs)
+	deletedVpcIPv4CIDRs := cache.vpcIPv4CIDRs.Difference(&newVpcIPv4CIDRs)
 
 	if !addedVpcIPv4CIDRs.IsEmpty() {
```

Comment:
Nit: Same `IsEmpty()` comment.  I think we should remove these checks.

Reply:
Since `IsEmpty()` is not used anywhere any more, I'll just remove it.